lang/sql.md

```diff
@@ -0,0 +1,120 @@
+---
+lang: SQL
+syntax: SQL
+title: SQL is a high-level declarative computer language.
+author: kunalcodes007
+---
+
+```SQL
+SELECT studentID, FirstName, LastName, FirstName + ' ' + LastName AS FullName
+FROM student;
+```
+
+ ## 🐍 What is SQL?
+ 
+SQL  is a domain-specific language used in programming and designed for managing data held in a relational database management system (RDBMS), or for stream processing in a relational data stream management system (RDSMS). It is particularly useful in handling structured data, i.e. data incorporating relations among entities and variables. SQL offers two main advantages over older read–write APIs such as ISAM or VSAM. Firstly, it introduced the concept of accessing many records with one single command. Secondly, it eliminates the need to specify how to reach a record, e.g. with or without an index.
+
+SQL became a standard of the American National Standards Institute (ANSI) in 1986, and of the International Organization for Standardization (ISO) in 1987. Since then, the standard has been revised to include a larger set of features. Despite the existence of standards, most SQL code requires at least some changes before being ported to different database systems.
+
+### 🤔 Why is SQL so popular?
+
+There are many reasons for the ubiquity of sql , including :
+
+- **Easy to Learn and Use**
+
+- **Mature and Supportive Community**
+
+- **Efficiency, Reliability, and Speed**
+
+- **First-choice Language**
+
+- **The Flexibility of sql Language**
+
+- **Use of sql in academics**
+
+## 📚 Why to use SQL?
+
+We got why `sql is so popular` at the moment above, but why should one learn how to implement it? Aside from the ease of use and versatility mentioned above, there are several good reasons to learn sql: 
+
+- **sql developers are in demand :** Across a wide range of fields, there is a [demand for those with sql skills]. If you’re looking to start or change your career, it could be a vital skill to help you. 
```

Comment:
```suggestion
- **SQL developers are in demand:** Across a wide range of fields, there is a demand for those with SQL skills. If you’re looking to start or change your career, it could be a vital skill to help you. 
```